files/alerts_logger.py

```diff
@@ -0,0 +1,150 @@
+
+    def __repr__(self):
+        return "TendrlAuth(token={})".format(self.__bearer_token)
+
+    def __call__(self, r):
+        """
+        Add Tendl Bearer Token into header of the request.
+        For full description, see requests documentation:
+        http://docs.python-requests.org/en/master/user/authentication/
+        """
+        headers = {
+            "Authorization": "Bearer {}".format(self.__bearer_token),
+            }
+        r.prepare_headers(headers)
+        return r
+
+
+def login(username, password, asserts_in=None):
+    """
+    Login Tendrl user.
+    Args:
+        username: name of user that is going logged in
+        password: password for username
+        asserts_in: assert values for this call and this method
+    Returns requests auth object (instance of TendrlAuth)
+    """
+    pattern = "login"
+    post_data = {"username": username, "password": password}
+    request = requests.post(
+        URL + pattern,
+        data=json.dumps(post_data))
+    token = request.json().get("access_token")
+    auth = TendrlAuth(token, username)
+    return auth
+
+
+def logout(auth, asserts_in=None):
+    """
+    Logout Tendrl user.
+    Args:
+        asserts_in: assert values for this call and this method
+        auth: TendrlAuth object (defines bearer token header)
+    """
+    pattern = "logout"
+    request = requests.delete(
+        URL + pattern,
+        auth=auth)
+    return request.json()
+
+
+class TendrlApi(object):
+    """ Common methods for Tendrl REST API.
+    """
+
+    def __init__(self, auth=None):
+        """
+        Args:
+            auth: TendrlAuth object (defines bearer token header), when auth is
+               None, requests are send without athentication header
+        """
+        # requests auth object with so called tendrl bearer token
+        self._auth = auth
+
+    def alerts(self):
+        """ Alerts REST API
+        Name:        "alerts",
+        Method:      "GET",
+        Pattern:     "alerts",
+        """
+        response = requests.get(
+            URL + "alerts",
+            auth=self._auth,)
+        return response.json()
+
+
+class TendrlFormatter(logging.Formatter):
+    """
+    This class can be used to create special formating of tendrl logs.
+    """
+    def format(self, record):
+        # TODO(fbalak): update this function to alter log format
+        result = super(TendrlFormatter, self).format(record)
+        return result
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("user", type=str, nargs='?')
+    args = parser.parse_args()
+    config = ConfigParser.RawConfigParser()
+    config.read("/etc/usmqe_alerts_logger_users.ini")
+    password = config.get(args.user, "password")
+    auth = login(args.user, password)
+    api = TendrlApi(auth=auth)
+    logging.basicConfig(level=os.environ.get("LOGLEVEL", "INFO"))
+    logger = logging.getLogger("tendrl-notifier.{}".format(args.user))
```

Comment:
The logs are in fact not from `tendrl-notifier`, but from *usmqe alerts_logger*, so I would reflect it in the logger name. To be easily possible to distinguish/grep logs from core *Tendrl* components and our own logs.